--- /home/app/unms/config.js	2019-01-15 02:37:31.000000000 -0800
+++ /home/app/unms/config.js	2019-02-01 21:41:17.277508639 -0800
@@ -117,6 +117,8 @@
 config.rabbitMQ = {
   host: process.env.UNMS_RABBITMQ_HOST || '127.0.0.1',
   port: toInteger(process.env.UNMS_RABBITMQ_PORT) || 5672,
+  user: process.env.UNMS_RABBITMQ_USER || '',
+  pass: process.env.UNMS_RABBITMQ_PASS || '',
   exchange: 'unms',
   notifyExchange: 'unms-notify',
 };
 