--- /home/app/unms/lib/message-hub/plugin.js	2018-01-24 01:49:56.000000000 -0800
+++ /home/app/unms/lib/message-hub/plugin.js	2019-02-01 21:44:45.967532396 -0800
@@ -15,9 +15,9 @@
  */
 function register(server, options) {
   const { logging } = server.plugins;
-  const { host, port, exchange: exchangeName } = options;
+  const { host, port, user, pass, exchange: exchangeName } = options;
   const bus = servicebus.bus({
-    url: `amqp://${host}:${port}`,
+    url: `amqp://${user}:${pass}@${host}:${port}`,
     exchangeName,
     correlator: new Correlator(),
     enableConfirms: true,
